documentdb/spring-data-azure-documentdb/src/test/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplateIT.java

```diff
@@ -89,7 +89,7 @@ public void testFindAll() {
 
     @Test
     public void testFindById() {
-        final Person result = dbTemplate.findById(TEST_PERSON.getId(), Person.class);
+        final Person result = dbTemplate.findById(Person.class.getSimpleName(), TEST_PERSON.getId(), Person.class);
```

Comment:
My personal preference is to test the findById API with no collection name parameter here, which can improve code coverage a little bit.